src/components/Footer/Footer.test.tsx

```diff
@@ -40,6 +40,23 @@ describe('<Footer />', () => {
             expect(cut.find('li.rvt-footer__aux-item > a')).toHaveLength(4);
             expect(cut.find('#privacy')).toHaveLength(1);
         });
+        it('should render copyright with current year', () => {
+            const copyrightText = "Copyright © " + new Date().getFullYear() + " The Trustees of Indiana University";
+            const cut = mount(<Footer />);
+            expect(cut.find('li.rvt-footer__aux-item').at(1).text()).toEqual(copyrightText);
+        });
+        it('should render copyright year argument specified', () => {
+            const year = 2020;
+            const copyrightText = "Copyright © " + year + " The Trustees of Indiana University";
+            const cut = mount(<Footer copyrightYear={year} />);
+            expect(cut.find('li.rvt-footer__aux-item').at(1).text()).toEqual(copyrightText);
+        });
+        it('should render copyright with current year when given a non number', () => {
+            const year = "2020 - 2021";
```

Comment:
Strings in JS are typically single quoted
```suggestion
            const year = '2020 - 2021';
```

Reply:
The `copyrightText` field in all of these tests should also use single quotes.